lib/src/switch/Switch.stories.tsx

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import { userEvent, within } from "@storybook/testing-library";
+import DxcSwitch from "./Switch";
+import { BackgroundColorProvider } from "../BackgroundColorContext";
+import Title from "../../.storybook/components/Title";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DarkContainer from "../../.storybook/components/DarkSection";
+
+export default {
+  title: "Switch",
+  component: DxcSwitch,
+};
+
+export const Chromatic = () => (
+  <>
+    <ExampleContainer>
+      <Title title="With label before" theme="light" level={4} />
+      <DxcSwitch label="Switch" />
+    </ExampleContainer>
+    <ExampleContainer>
```

Comment:
This test is not necessary as we are already testing the label before the rest of tests. We can avoid repeated tests if we delete it.